tethys_cli/db_commands.py

```diff
@@ -22,7 +24,20 @@ def add_db_parser(subparsers):
 
     # Setup db commands
     db_parser.add_argument('command',
-                           help='DB command to run.',
+                           help='Performs operations on the Tethys Database.\n'
+                                '  * init - Creates a new, locally running PostgreSQL database server.\n'
+                                '  * start - Starts the local database server.\n'
+                                '  * stop - Stops the local database server.\n'
+                                '  * status - Gets the current status of the local database server.\n'
+                                '  * create - Creates the Tethys Portal database on the database server configured '
+                                'in the `portal-config.yml` (this could be local or remote).\n'
+                                '  * migrate - Runs migrations on the Tethys Portal database.\n'
+                                '  * createsuperuser - Creates a Tethys Portal superuser account.\n'
+                                '  * configure - A shortcut for running: init, start, create, migrate, and '
+                                'createsuperuser. Note: if the database server that is configured in '
+                                '`portal_config.yml` is remote then the init and start commands are skipped.\n'
```

Comment:
Technically a docker container or system database could be running locally, but you wouldn't want to run the init or start  steps. For clarity, this should probably say something to the effect of "if the DIR parameter is not defined in your DATABASES.default configuration in portal_config.yml then the init and start commands are skipped."